src/testController.ts

```diff
@@ -91,37 +98,44 @@ export class TestController {
 
     response.forEach((res) => {
       const [_, name, command, location] = res.command!.arguments!;
-      const testItem: vscode.TestItem = this.testController.createTestItem(
-        name,
-        name,
-        uri
-      );
+      let testItem;
```

Comment:
Getting `Variable 'testItem' implicitly has type 'any' in some locations where its type cannot be determined.` locally from TS lsp.